code/game/objects/items/toys/toys.dm

```diff
@@ -472,13 +472,31 @@
 		user.visible_message(SPAN_NOTICE("[user] hugs [src]! How cute! "), \
 							SPAN_NOTICE("You hug [src]. Dawwww... "))
 		last_hug_time = world.time + 50 //5 second cooldown
+		playsound(user, "plush", 25, TRUE)
 
 /obj/item/toy/farwadoll/pred
 	name = "strange plush doll"
 	desc = "A plush doll depicting some sort of tall humanoid biped..?"
 	w_class = SIZE_TINY
 	icon_state = "predplush"
 
+/obj/item/toy/plushie_cade
+	name = "plushie barricade"
+	desc = "Great for squeezing whenever you're scared. Or lightly hurt. Or in any other situation."
+	icon_state = "plushie_cade"
+	item_state = "plushie_cade"
+	w_class = SIZE_SMALL
+	var/last_hug_time
```

Comment:
please check if we have cooldown macros (`DECLARE_COOLDOWN` [maybe it's backwards, I don't remember] and etc.), and change this to that if so

also, codedoc comment

Reply:
we do btw